packages/common/src/api/collection.ts

```diff
@@ -42,9 +42,12 @@ const collectionApi = createApi({
         { ids, currentUserId }: { ids: ID[]; currentUserId?: Nullable<ID> },
         { audiusSdk }
       ) => {
+        const id = ids.filter((id) => id && id !== -1).map((id) => Id.parse(id))
```

Comment:
Could use > 0 to be overly cautious, but probably won't come up

Reply:
Id is an odd name for a list

Reply:
It's because the query string parameter is `id` and gets expanded into that odd list syntax.